cms/envs/common.py

```diff
@@ -254,9 +254,12 @@
     # Enable the courseware search functionality
     'ENABLE_COURSEWARE_INDEX': False,
 
-    # Enable content libraries search functionality
+    # Enable content libraries (modulestore) search functionality
     'ENABLE_LIBRARY_INDEX': False,
 
+    # Enable content libraries (blockstore) indexing
+    'ENABLE_CONTENT_LIBRARY_INDEX': True,
+
```

Comment:
I agree to keep this disabled for now (even for devstack I'd say - would be confusing if devstack and production behaved differently by default). We can enable this down the line when the API gets more stable.

Reply:
OTOH, an issue I see is that it would return the "limited" set of metadata, with `num_blocks`, `last_published`, `has_unpublished_changes` and `has_unpublished_deletes` values in the `/libraries` API as `null`, which might break any existing clients.

Reply:
For that compatibility, we can revert and enable it again.

Reply:
> OTOH, an issue I see is that it would return the "limited" set of metadata, with num_blocks, last_published, has_unpublished_changes and has_unpublished_deletes values in the /libraries API as null, which might break any existing clients.

@mavidser Ah, right. So the only backwards-compatible thing to do is to set it to True, right?

> For that compatibility, we can revert and enable it again.

Sorry, I'm not sure I follow what you mean here.

Reply:
> Sorry, I'm not sure I follow what you mean here.

Ah yes, that _is_ quite unclear. I'd updated the code to disable indexing before posting that comment, so was presenting the option to enable it again to preserve compatibility for clients. I'm also considering backporting some changes from #24510 so that this can be merged more safely.

Reply:
@mavidser Gotcha. Now that I think about it... am I correct that LabXChange is the only client of this API currently? If so, I would leave to you and the other folks at OpenCraft to decide what the safest way to roll this out is based on the aspects of this API that you count on :smile: 

And feel free to pull in anything you feel is prudent to from #24510 ; if you do, let me know when you're ready for another round of review.